blotoutSDK/src/main/java/com/analytics/blotout/repository/EventRepository.kt

```diff
@@ -34,15 +34,7 @@ class EventRepository(private var secureStorage: SharedPreferenceSecureVault) {
             val deviceAndAppFraudController = DeviceAndAppFraudController(context)
             meta.sdkv = "".getVersion()
             meta.tzOffset = DateTimeUtils().getCurrentTimezoneOffsetInMin()
-            meta.userIdCreated = CommonUtils().getUserBirthTimeStamp()
-            meta.plf = Constant.BO_Android_All
-            meta.osv = Build.VERSION.RELEASE
-            meta.appv = context.getVersion()
```

Comment:
is appv something that we get in user agent. If not let's keep this one

Reply:
user agent wont provide app version detail ? To get appv back , I have dropped comment on SDK document as well to add it back.

Reply:
cool, adding it back to docs